content/psychologists-recommend/preparing-to-exam.md

```diff
@@ -0,0 +1,56 @@
+title: Как правильно подготовить себя к сдаче экзамена
+slug: preparing-to-exam
```

Comment:
Replace `slug` name with `preparing-for-exams`. File name should be renamed as `preparing_for_exams`.